composer.json

```diff
@@ -5,20 +5,30 @@
     "license": "GPL-2.0-or-later",
     "require": {
         "composer/installers": "^1.2",
-        "drupal/ajax_comments": "1.x-dev#4b0c423",
-        "drupal/ckeditor_mentions": "2.x-dev#272a28e7",
+        "dealerdirect/phpcodesniffer-composer-installer": "^0.7.1",
```

Comment:
dealerdirect/phpcodesniffer-composer-installer should not be in require but in require-dev

Reply:
@heyyo-droid moved in require-dev section.
Thank you!